production.ini

```diff
@@ -106,28 +106,51 @@ comps_url = https://git.fedorahosted.org/comps.git
 ##
 ## Mirror settings
 ##
+# file_url: Used in the repo metadata to set RPM URLs.
 file_url = https://download.fedoraproject.org/pub/fedora/linux/updates
-master_repomd = http://download.fedora.redhat.com/pub/fedora/linux/updates/%d/i386/repodata/repomd.xml
-fedora_master_repomd = http://download.fedora.redhat.com/pub/fedora/linux/updates/%d/i386/repodata/repomd.xml
-fedora_epel_master_repomd = http://download.fedora.redhat.com/pub/epel/%d/i386/repodata/repomd.xml
+
+# {release}_{request}_master_repomd: This is used by the masher to determine when a
+#     primary architecture push has been synchronized to the master mirror for a given release and
+#     request. The masher will verify that the checksum of repomd.xml at the master URL matches the
+#     expected value, and will poll the URL until this test passes. Substitute release and request
+#     for each release id (replacing -'s with _'s) and request (stable, testing). Used for the
+#     arches listed in {release}_{version}_primary_arches when it is defined, else used for all
+#     arches. You must put two %s's in this setting - the first will be replaced with the release
+#     version and the second will be replaced with the architecture.
+fedora_stable_master_repomd = http://download01.phx2.fedoraproject.org/pub/fedora/linux/updates/%s/%s/repodata/repomd.x
```

Comment:
I fixed these missing "ml"'s in https://github.com/fedora-infra/bodhi/pull/1371. Thanks for noting that!